vote-signer/src/rpc.rs

```diff
@@ -132,13 +129,20 @@ fn verify_signature(sig: &Signature, pubkey: &Pubkey, msg: &[u8]) -> Result<()>
     }
 }
 
+pub trait VoteSignerInterface {
```

Comment:
Can we rename that to just `VoteSigner`?  There's no precedent in Rust or our codebase for including a suffix like `Interface`.  There is precedent for naming it after the trait's primary method. In this case, that'd make it `Sign`, but I can see how that'd be confusing with this particular name.